glide.yaml

```diff
@@ -102,3 +102,6 @@ import:
   subpackages:
   - unix
   - windows
+- package: github.com/moby/buildkit
+  version: da2b9dc7dab99e824b2b1067ad7d0523e32dd2d9
+  repo: https://github.com/dmcgowan/buildkit.git
```

Comment:
Why are we using a fork instead of upstream?

Reply:
It's the version that the docker fork specified.  When I didn't pin it, glide wasn't able to resolve compatible versions of its dependencies.

Reply:
@nalind please add a comment to the `glide.yaml` that these pins resolve dependencies in the `openshift/moby-moby` branch.

Reply:
Added.